This call to `pretty_inspect` causes Avram to not compile on Windows

https://github.com/luckyframework/lucky/blob/921436f3b1d83ed95fd7e7c3d6ff5f90d337b0d9/src/lucky/errors.cr#L252

Specifically calling `require "lucky"` from this line https://github.com/luckyframework/avram/blob/b04ba2d4c130d6ddaf98069bfdedbf8a2b7b935a/src/lucky.cr#L1 will throw a compile error

```
In D:\a\_temp\crystal-latest-true-undefined\src\openssl\ssl\socket.cr:150:33

 150 | raise OpenSSL::SSL::Error.new(@ssl, bytes, "SSL_write")
                                 ^--
Error: Error: instance variable '@underlying_eof' of OpenSSL::SSL::Error must be Bool, not Nil
Error: Process completed with exit code 1.
```

This traces down to this line

```
In lib\lucky\src\lucky\errors.cr:252:54

 252 | Expected subdomain matcher(s): #{@expected.pretty_inspect}
                                                  ^-------------
Error: Error: instantiating '(Array(Regex | String) | Array(Regex) | Array(String) | Bool | Regex | String)#pretty_inspect()'
```

Removing the call to `pretty_inspect` "fixes" the issue and lets specs run on Avram... I haven't been able to reduce this outside of Avram. It's possible there's an issue somewhere else and this is a false positive, but for now I think we can just remove it until we can come up with a proper fix. It's most likely a Crystal bug on windows.

--- a/src/lucky/errors.cr
+++ b/src/lucky/errors.cr
@@ -249,7 +249,7 @@ module Lucky
         "Expected request to have a subdomain but did not find one."
       else
         <<-MESSAGE
-          Expected subdomain matcher(s): #{@expected.pretty_inspect}
+          Expected subdomain matcher(s): #{@expected}
           Did not match host: #{@host}
         MESSAGE
       end

